### DISCUSSION TITLE
caching a texture at a lower resolution than its original version

### DISCUSSION THREAD

**Discussion by 10100010010001001010** [axmolengine/axmol#1337] [General]
Is there a way to caching in video memory not the original texture, but its compressed version? I draw textures in high resolution, it’s more convenient, but storing such a texture is resource-intensive

**Comment by DelinWorks** [comment]
use the image class to load image data and before assigning a texture with an image, write an algorithm to make your image smaller, you can access the image buffer pointer which consists of 32bit image data stored linearly in cpu memory, copy & edit that buffer and make a new image out of it and give it to a texture.

**Reply by DelinWorks** [reply]
if you're talking about straight compression then axmol supports astc

**Reply by 10100010010001001010** [reply]
thanks
